airbyte-cdk/python/unit_tests/sources/streams/concurrent/test_adapters.py

```diff
@@ -122,6 +123,8 @@ def test_stream_partition(transformer, expected_records):
             message='slice:{"partition": 1}',
         ),
     )
+    for record in expected_records:
```

Comment:
Why did we need to add this?

Reply:
we need to add an initiated partition to the records in `expected_records`